core/domain/skill_validators.py

```diff
@@ -137,8 +137,8 @@ class SkillCommitLogEntryModelValidator(
 
     @classmethod
     def _get_model_id_regex(cls, item):
-        # Valid id: [skill/rights]-[skill_id]-[skill_version].
-        regex_string = '^(skill|rights)-%s-\\d+$' % (
+        # Valid id: [skill]-[skill_id]-[skill_version].
+        regex_string = '^(skill)-%s-\\d+$' % (
```

Comment:
Could you also check stories and questions validators that `rights` is also removed there?

Reply:
Yup, they are removed